modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/identity/mgt/AccountLockEnabledTestCase.java

```diff
@@ -92,29 +93,21 @@ public void testInit() throws Exception {
     @Test(groups = "wso2.is", description = "Check whether the user account lock successfully")
     public void testSuccessfulLockedInitially() {
         try {
-            usmClient.addUser(testLockUser1, testLockUser1Password, new String[]{"admin"}, new ClaimValue[0], null, false);
+            testLockUserId = addAdminUser(testLockUser1,testLockUser1Password, null);
```

Comment:
```suggestion
            testLockUserId = addAdminUser(testLockUser1, testLockUser1Password, null);
```

Reply:
Fixed with https://github.com/wso2/product-is/pull/15906/commits/58e6cec24ce802db3f770b7bbac811a582bb02c2